### PR TITLE
use ansible module

### PR DESCRIPTION
**Pull request by astail**

### CODE REVIEW

File: ansible/roles/cdh_common/tasks/main.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@
     - topology.sh
 
 - name: run update-alternatives to install hadoop configuration
-  command: update-alternatives --install /etc/hadoop/conf hadoop-conf /etc/hadoop/conf.mycluster 50
+  alternatives: name=hadoop-conf link=/etc/hadoop/conf path=/etc/hadoop/conf.mycluster
```
Comment (qilab) [inline ansible/roles/cdh_common/tasks/main.yml]: `alternatives` is an extra module now,
http://docs.ansible.com/ansible/alternatives_module.html#this-is-an-extras-module
and I think it is enough to use it after it becomes core module.

cf (in Japanese):
http://onigra.github.io/blog/2014/12/11/ansible-extra-and-alternatives-module/

File: ansible/roles/cdh_common/tasks/main.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@
   yum: name=hadoop-conf-pseudo state=present
 
 - name: copy /etc/hadoop/conf.pseudo to /etc/hadoop/conf.mycluster
-  command: cp -R -p /etc/hadoop/conf.pseudo /etc/hadoop/conf.mycluster creates=/etc/hadoop/conf.mycluster
+  copy: src=/etc/hadoop/conf.pseudo dest=/etc/hadoop/conf.mycluster directory_mode=yes remote_src=True
```
Comment (qilab) [inline ansible/roles/cdh_common/tasks/main.yml]: http://docs.ansible.com/ansible/copy_module.html

`directory_mode`:

> When doing a recursive copy set the mode for the directories. 

`remote_src`:

> Currently remote_src does not support recursive copying.

So, you can't use `directory_mode=yes` and `remote_src=True` at the same time.

Comment (astail) [inline ansible/roles/cdh_common/tasks/main.yml]: sorry.
Thank you very much  Review.

**Comment by qilab** [comment]
:+1: 
Thanks!

**Pull request merged**